src/pages/dashboardOverview/components/activeProductsSection/components/ActiveProductCardContainer.tsx

```diff
@@ -23,7 +24,8 @@ export default function ActiveProductCardContainer({
   const { t } = useTranslation();
   const { invokeProductBo } = useTokenExchange();
 
-  const [openEnvironmentModal, setOpenEnvironmentModal] = useState<boolean>(false);
+  const [openCustomEnvInteropModal, setopenCustomEnvInteropModal] = useState<boolean>(false);
```

Comment:
setter function name should be in camelcase

Reply:
done